ddr/lib/ddr-macros/CMakeLists.txt

```diff
@@ -19,14 +19,14 @@
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 ###############################################################################
 
-add_library(omr_ddr_macros
+add_library(ddr_macros
```

Comment:
The changes to this file seem unrelated and unnecessary.

Reply:
these changes are to bring things inline with the makefile target names (which do not have the `omr_` prefix

Reply:
I think these changes would have been better isolated in a separate pull request.